CanvasPlusPlayground/Views/PeopleCommonView.swift

```diff
@@ -19,20 +19,22 @@ struct PeopleCommonView: View {
                 statusLabel
             }
 
-            ForEach(courses, id: \.id) { course in
+            ForEach(commonCourses, id: \.id) { course in
                 Text(course.name ?? "")
             }
         }
         .task {
             loading = true
-            self.courses = await peopleManager.fetchAllClassesWith(userID: user.id!)
+            await peopleManager.fetchAllClassesWith(userID: user.id!) {
```

Comment:
we shouldve addressed this earlier, but can we prevent force unwrapping here? e.g. a guard